src/quacc/utils/coords.py

```diff
@@ -0,0 +1,26 @@
+"""Utilities for working with atomic coordinates."""
+
+from __future__ import annotations
+
+import copy
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+if TYPE_CHECKING:
+    from ase import Atoms
+
+
+def perturb(mol: Atoms, vector: list[list[float]], scale: float):
+    mol_copy = copy.deepcopy(mol)
```

Comment:
Can you use `quacc.atoms.core.copy_atoms()` here? There are some rare edge cases where a `deepcopy()` can yield a crash.